src/XIVLauncher.Core/Components/MainPage/MainPage.cs

```diff
@@ -144,7 +145,7 @@ private void ProcessLogin(LoginAction action)
 
             App.Settings.IsAutologin = this.loginFrame.IsAutoLogin;
 
-            var result = await Login(loginFrame.Username, loginFrame.Password, loginFrame.IsOtp, loginFrame.IsSteam, false, action).ConfigureAwait(false);
+            var result = await Login(loginFrame.Username, loginFrame.Password, loginFrame.IsOtp, loginFrame.IsSteam, loginFrame.IsFreeTrial, false, action).ConfigureAwait(false);
```

Comment:
These arguments are in the wrong order. If you look below to the Login() Task, `doingAutoLogin` follows `isSteam`, and then `IsFreeTrail`. Here, the `loginFrame.IsFreeTrial` is being passed as the `doingAutoLogin`, and `isFreeTrial` is always false.